### PR TITLE
[Screenshots] added Frontend and admin frontend part

### PR DESCRIPTION
**Pull request by yvesh**
Basic screenshot functionality

* Screenshots for frontend
* And Frontend as Administrator
* Changed Resolution to 1366x768
* Tabs are now working (#images (check the href of the tab as target))

### CODE REVIEW

**Comment by astridx** [comment]
I tested this new feature successfully.
1. I applied the branch: git fetch upstream pull/119/head:screenshots
2. I edited the file /tests/codeception/screenshots.suite.yml
3. I run the command 
     tests/codeception/vendor/bin/codecept run screenshots
4. I found the screen shots in the folder screenshots of my Joomla! root


Also it worked in this case with the parameter

->arg('--env ' . $opts['env'])

I would change this to 

->env($opts['env'])

File: RoboFile.php [hunk 1]
```diff
@@ -599,11 +599,7 @@ private function isWindows()
 	/**
 	 * Return the correct path for Windows
 	 *
-<<<<<<< HEAD
```
Comment (810) [inline RoboFile.php:602]: fix conflicts

Comment (810) [inline RoboFile.php]: fix conflicts

Comment (yvesh) [reply RoboFile.php:602]: Jelle i fixed them there :) Or what do you mean?

**Pull request merged**